### PR TITLE
feat: add logger

### PR DESCRIPTION
**Pull request by marvinmarnold**
**Why**
It is helpful to have verbose logs that are consistent across all services. 

**How**
This logger has been copied from [hm-config](https://github.com/NebraLtd/hm-config/blob/ca7b5e3e05041b9fdaaa238e3240c1f1e38f1cfd/gatewayconfig/logger.py).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
...
+logger = get_logger(__name__)
+logger.debug("message to log")
+```
+
+## Testing
+
+To run tests: 
+
+```bash
+pip install -r test-requirements
```
Comment (vpetersson) [inline README.md:20]: Needs to be:

* `pip install -r requirements.txt`
* `pip install -r test-requirements.txt`

File: README.md [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+PYTHONPATH=./ pytest
```
Comment (vpetersson) [inline README.md:21]: Don't think the explicit `PYTHONPATH` is required here.

**Review by vpetersson — APPROVED** [review]
Accepted with minor comment.

File: README.md (hunk 2 above)
Comment (marvinmarnold) [reply README.md:21]: I'm unsure if this is a best practice but with the code as-is now, you'll get an error if just trying to `pytest`. This type of thing is a problem whenever we have the test code in a separate directory/module. In other repos, we get around it by [modifying the path](https://github.com/NebraLtd/hm-diag/blob/master/hw_diag/tests/test_config_search_param.py#L4) from the test code, which I think is worse.

```
✗ pytest
========================================== test session starts ===========================================
platform linux -- Python 3.7.3, pytest-6.2.5, py-1.10.0, pluggy-1.0.0
rootdir: /home/user/code/nebra/hm-pyhelper
collected 10 items / 1 error / 9 selected                                                                

================================================= ERRORS =================================================
________________________ ERROR collecting hm_pyhelper/tests/utils/test_logger.py _________________________
ImportError while importing test module '/home/user/code/nebra/hm-pyhelper/hm_pyhelper/tests/utils/test_logger.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/home/user/.pyenv/versions/3.7.3/lib/python3.7/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/utils/test_logger.py:2: in <module>
    from utils.logger import get_logger, _log_format
```

File: README.md (hunk 1 above)
Comment (marvinmarnold) [reply README.md]: Thanks, fixed: https://github.com/NebraLtd/hm-pyhelper/pull/14/commits/9d9d1d8728fb1f6c216d69a0aecb23ea14cdae27

File: README.md (hunk 2 above)
Comment (vpetersson) [reply README.md:22]: `pwd` should be in PYTHONPATH, but this is not a big deal.

**Comment by vpetersson** [comment]
LGTM, but i think you need to merge in master and bump up the version.

File: setup.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 setup(
     name='hm_pyhelper',
-    version='0.6.1',
+    version='0.6.3',
```
Comment (marvinmarnold) [inline setup.py:6]: This had to be bumped twice because of this: https://github.com/NebraLtd/hm-pyhelper/issues/20

**Comment by vpetersson** [comment]
@marvinmarnold FYI - to release you need to create a git tag for the release (e.g. `v0.6.3`), push this and then generate a release in GitHub. The new release will then be built and pushed to PyPi.

**Pull request merged**